src/EventStore.Client/ResolvedEvent.cs

```diff
@@ -1,22 +1,56 @@
 #nullable enable
 namespace EventStore.Client {
+	/// <summary>
+	/// A structure representing a single event or a resolved link event.
+	/// </summary>
 	public readonly struct ResolvedEvent {
+		/// <summary>
+		/// If this <see cref="ResolvedEvent"/> represents a link event, the <see cref="OriginalEvent"/>
+		/// will be the link, otherwise it will be the event.
```

Comment:
```suggestion
		/// If this <see cref="ResolvedEvent"/> represents a link event, the <see cref="Event"/>
		/// will be the resolved link event, otherwise it will be the single event.
```

@thefringeninja is the above suggestion correct?